wpimath/src/main/native/include/units/base.h

```diff
@@ -1992,6 +1992,7 @@ namespace units
 		 * @param[in]	rhs unit to copy.
 		 */
 		template<class UnitsRhs, typename Ty, template<typename> class NlsRhs>
+			requires (traits::is_unit<UnitsRhs>::value && traits::is_unit<Units>::value && traits::is_convertible_unit<UnitsRhs, Units>::value)
```

Comment:
```suggestion
			requires traits::is_unit_v<UnitsRhs> && traits::is_unit_v<Units> && traits::is_convertible_unit_v<UnitsRhs, Units>
```

Reply:
Ah, forgot that primary expressions don't need parentheses in `requires` clauses. (And I just copy-pasted the conditions in the `static_assert`s, but it's definitely nicer to use `_v` instead of `::value`.)